packages/parser-typescript-config/src/index.ts

```diff
@@ -0,0 +1 @@
+module.exports = { javascript: require('./typescript-config') };
```

Comment:
I think you meant `typescript-config` here.

Reply:
good catch!